src/test/java/org/broadinstitute/consent/http/service/dao/DarCollectionServiceDAOTest.java

```diff
@@ -372,50 +408,59 @@ void testCreateElectionsForDarCollectionAfterCancelingEarlierElectionsAsChair()
     assertEquals(1, createdElections.stream().filter(e -> e.getElectionType().equals(
         ElectionType.RP.getValue())).count());
 
-    // Make sure we did not create elections for the DAR/Dataset that the chair does not have access to.
-    List<Election> nonCreatedElections =
-        electionDAO.findLastElectionsByReferenceIds(List.of(dar2.getReferenceId()));
-    assertTrue(nonCreatedElections.isEmpty());
+    //create progress report
+    createProgressReportFromDAR(dar);
+    collection = darCollectionDAO.findDARCollectionByCollectionId(collection.getDarCollectionId());
+    List<String> createdElectionsForProgressReport = serviceDAO.createElectionsForDarByUser(chair.get(), collection.getMostRecentDar());
+    assertFalse(createdElectionsForProgressReport.isEmpty());
+    assertEquals(1, createdElectionsForProgressReport.size());
   }
 
+  private DataAccessRequest createProgressReportFromDAR(DataAccessRequest dar) {
+    String referenceId = UUID.randomUUID().toString();
+    dataAccessRequestDAO.insertProgressReport(
+        dar.getId(),
+        dar.getCollectionId(),
+        referenceId,
+        dar.getUserId(),
+        dar.getData());
+    DataAccessRequest progressReport = dataAccessRequestDAO.findByReferenceId(referenceId);
+    dar.getDatasetIds().forEach(datasetId -> {
+      dataAccessRequestDAO.insertDARDatasetRelation(referenceId, datasetId);
+    });
```

Comment:
No `{}` needed for a one line lambda
```suggestion
    dar.getDatasetIds().forEach(datasetId ->
      dataAccessRequestDAO.insertDARDatasetRelation(referenceId, datasetId));
```